src/components/Banner/Banner.scss

```
@@ -221,7 +166,7 @@ $secondary-action-horizontal-padding: 1.5 * spacing(tight);
     padding-right: spacing();
   }
 
-  @include banner-variants;
+  @include banner-variants(false);
```

```suggestion
  @include banner-variants($in-page: false);
```
Might be a bit clearer to use the named var